ros2topic/ros2topic/verb/echo.py

```diff
@@ -41,6 +54,15 @@ def add_arguments(self, parser, cli_name):
             '--csv', action='store_true',
             help='Output all recursive fields separated by commas (e.g. for '
                  'plotting)')
+        parser.add_argument(
+            '--full-length', action='store_true',
```

Comment:
I think it would be good to have a short flag for this, maybe `-f` or `-a`. @ros2/team what does everyone else think?